### PR TITLE
Make byline avatars round, don't mess with size

### PR DESCRIPTION
**Pull request by benlk**
## Changes

Without this change:

screen shot 2016-12-13 at 6 53 46 pm

With this change:

screen shot 2016-12-13 at 6 51 31 pm

Copying these styles into Largo: https://github.com/INN/theme-rivard-report/blob/c7328ca384a37c03b83245e6a9a26892749a4d17/less/defaults.less#L561-L563

## Why

Because of https://app.asana.com/0/145571381285032/228422685302617 for Public Source and the resulting Largo issue #1386.

### CODE REVIEW

**Comment by rclations** [comment]
see my note in #1386 - we shouldn't be making any changes besides bug fixes to point releases - this should instead be considered for the next feature release.

closing so this doesn't accidentally get merged into master

**Comment by aschweigert** [comment]
This is needed for publicsource so i'm 👍 on just merging to master unless there are serious objections

**Comment by aschweigert** [comment]
There is one more odd issue, reported by publicsource where if a non-square image is uploaded as an avatar, it seems to get weirdly distorted (because the current max-width/height styles force it to be square event if the avatar-32 crop returned here is not square). Let's try to address that and then merge. I'd also be pro-making the image slightly larger (something like rivard would be better than 19px which seems to be the default here)

**Comment by aschweigert** [comment]
see: https://app.asana.com/0/145571381285032/232572519703827

File: inc/byline_class.php [hunk 1]
```diff
@@ -103,7 +103,7 @@ function avatar() {
 		} else {
 			$author_email = get_the_author_meta( 'email', $this->author_id );
 			if ( $this->author->type == 'guest-author' && get_the_post_thumbnail( $this->author->ID ) ) {
-				$output = get_the_post_thumbnail( $this->author->ID, array( 32,32 ) );
+				$output = get_the_post_thumbnail( $this->author->ID, array( 60,60 ) );
```
Comment (rclations) [inline inc/byline_class.php]: 32x32 image size doesn't exist in Largo, 60x60 thumbnails are defined in 
https://github.com/INN/Largo/blob/56c40405990d7693fadfa082196b5339963d445b/functions.php#L256

**Comment by rclations** [comment]
works for me - my only hesitation was creating visual changes on a maintenance release.
Ben's code looks good to me - @aschweigert could you test my commit to make sure that it's not still loading in the full author images?

nice side affect is this fix will cut load time a bit on article pages

**Comment by rclations** [comment]
works for me - my only hesitation was creating visual changes on a maintenance release.
Ben's code looks good to me - @aschweigert could you test my commit to make sure that it's not still loading in the full author images?

nice side affect is this fix will cut load time a bit on article pages

**Comment by benlk** [comment]
Yep, it's no longer loading the full author image, but instead the 60x60 version, with srcset fallbacks:

```html
<span class="author vcard" itemprop="author">
<img width="60" height="60" 
    src="http://publicsource.invalid/wp-content/uploads/2016/08/20150916-eholmberg-kelly-60x60.jpg"
    class="attachment-60x60 size-60x60 "
    alt="20150916-eholmberg-kelly"
    srcset="
        http://publicsource.invalid/wp-content/uploads/2016/08/20150916-eholmberg-kelly-60x60.jpg 60w,
        http://publicsource.invalid/wp-content/uploads/2016/08/20150916-eholmberg-kelly-140x140.jpg 140w, 
        http://publicsource.invalid/wp-content/uploads/2016/08/20150916-eholmberg-kelly-32x32.jpg 32w, 
        http://publicsource.invalid/wp-content/uploads/2016/08/20150916-eholmberg-kelly-50x50.jpg 50w, 
        http://publicsource.invalid/wp-content/uploads/2016/08/20150916-eholmberg-kelly-64x64.jpg 64w, 
        http://publicsource.invalid/wp-content/uploads/2016/08/20150916-eholmberg-kelly-96x96.jpg 96w, 
        http://publicsource.invalid/wp-content/uploads/2016/08/20150916-eholmberg-kelly-128x128.jpg 128w
    " 
    sizes="(max-width: 60px) 100vw, 60px"
>
<a class="url fn n" href="http://publicsource.invalid/author/authoreric-holmberg/" title="Read All Posts By Eric Holmberg" rel="author">Eric Holmberg</a>
<span class="job-title"><span class="comma">,</span> Reporter | PublicSource</span>
</span>
```

(On a site with Co-Authors enabled, that is)

`max-height` is now set to 30px, and all images in the byline have this style applied so that we hit both coauthors and non-coauthors.

screen shot 2017-01-03 at 3 15 28 pm

Here's [Pesticide](https://github.com/mrmrs/pesticide) showing that it's all vertically centered: 

screen shot 2017-01-03 at 3 15 47 pm

And at normal zoom levels: 

screen shot 2017-01-03 at 3 17 53 pm

**Comment by aschweigert** [comment]
lgtm. @benlk can you resolve the conflicts and merge?

**Pull request merged**